tests/test_basic_ops_dict.py

```diff
@@ -75,6 +75,23 @@ def test_getattr_dict() -> None:
     assert {"b": 1} == c.a
 
 
+@mark.parametrize(
+    "cfg, struct",
+    [
+        (OmegaConf.create({"name": "alice", "age": 1}), False),
+        (OmegaConf.create({"name": "alice", "age": 1}), True),
+        (OmegaConf.structured(User(name="alice", age=1)), False),
+    ],
+)
```

Comment:
Try to parameterize like this. This will give all 4 combinations.
```python
@mark.parametrize("struct" : [True, False])
@mark.parametrize("cfg" : [
        param(OmegaConf.create({"name": "alice", "age": 1}), id="dict"),
        param(OmegaConf.structured(User(name="alice", age=1)), id="structured_config"),
])
```

Reply:
nice!